languages/csharp/config.json

```diff
@@ -50,11 +50,10 @@
         "uuid": "978bcc16-0c49-4328-92e9-79f6204ce350",
         "concepts": ["properties"],
         "prerequisites": [
-          "numbers",
-          "floating-point-numbers",
-          "classes",
-          "enums",
-          "exceptions"
+          "strings-basic",
```

Comment:
Could you revert the changes in this file?

Reply:
Oh, and update the entry of the `enums` exercise to match the concepts suggested elsewhere in this review?

Reply:
I don't exactly know how to revert the changes to this file. I'm still looking into it.

Reply:
I think this should ought to fix it: https://github.com/exercism/v3/pull/864/commits/51092cdb6589c78561759829a1b38a9972e3a309

Reply:
Yes, that worked.